we.go

```diff
@@ -562,33 +568,24 @@ func marshalLang(rv map[string]interface{}, r *resp) error {
 }
 
 func main() {
-	var numdays = 3
-
 	if len(config.APIKey) == 0 {
 		log.Fatal("No API key specified. Setup instructions are in the README.")
 	}
 	params = append(params, "key="+config.APIKey)
 
-	for _, arg := range os.Args[1:] {
-		if v, err := strconv.Atoi(arg); err == nil && len(arg) == 1 {
-			numdays = v
-		} else {
-			config.City = arg
-		}
-	}
```

Comment:
That is correct. The condition has been used before and is necessary to differentiate between the  two possible arguments, when no flags are used.